{"name": "test_mult[小数乘于字符串异常用例]", "status": "passed", "parameters": [{"name": "a", "value": "22.0"}, {"name": "b", "value": "'11'"}, {"name": "expect", "value": "'TypeError'"}], "start": 1603377912318, "stop": 1603377912318, "uuid": "6b9f8b3c-9d15-4f74-ac48-c0134302f6a0", "historyId": "b01049d1adef3b233619a2f1a6734d9c", "testCaseId": "5ba7137ceb70f9037bb50d3c2be4f3da", "fullName": "test_calc1.TestCalcTaskOne#test_mult", "labels": [{"name": "tag", "value": "run(order=4)"}, {"name": "tag", "value": "mult"}, {"name": "suite", "value": "test_calc1"}, {"name": "subSuite", "value": "TestCalcTaskOne"}, {"name": "host", "value": "lingz-mac.lan"}, {"name": "thread", "value": "2038-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc1"}]}